reana_server/openapi_connections/reana_workflow_controller.json

```diff
@@ -249,6 +249,79 @@
         },
         "summary": "Creates a new yadage workflow from a specification file."
       }
+    },
+    "/api/cwl/spec": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "description": "This resource is expecting a JSON cwl specification.",
+        "operationId": "run_cwl_workflow_from_spec",
+        "parameters": [
+          {
+            "description": "Required. Organization which the worklow belongs to.",
+            "in": "query",
+            "name": "organization",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "description": "Required. UUID of workflow owner.",
+            "in": "query",
+            "name": "user",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "description": "JSON object including workflow parameters and workflow specification in JSON format (`cwlschemas.load()` output) with necessary data to instantiate a cwl workflow.",
+            "in": "body",
+            "name": "workflow",
+            "required": true,
+            "schema": {
+              "properties": {
+                "parameters": {
+                  "description": "Workflow inputs",
+                  "type": "object"
+                },
+                "workflow_spec": {
+                  "description": "cwl specification filename",
+                  "type": "string"
```

Comment:
```json
"type": "object"
```
Since we are sending a `json` object from `reana-client`
Moreover, the process to edit this `openapi.json` files should be the following:
- You change API on `reana-workflow-controller` (or other component you want to consume) and remember to amend `docstrings`  accordingly.
- You run `./run-tests.sh` on `reana-workflow-controller` (or other component) which will generate the new specification file according to your changes.
- Copy the brand new specification file to `reana-client`'s `openapi_connections` directory.

This way you will always have the last tested API